packages/core/src/utils/validate-sign-in-experience.test.ts

```diff
@@ -39,3 +43,36 @@ describe('validate terms of use', () => {
     }).toMatchError(new RequestError('sign_in_experiences.empty_content_url_of_terms_of_use'));
   });
 });
+
+describe('validate sign-in methods', () => {
+  test('should not throw if without sign-in methods', () => {
+    expect(() => {
+      // eslint-disable-next-line unicorn/no-useless-undefined
+      validateSignInMethods(undefined);
```

Comment:
wondering why put an `undefined` here?

Reply:
- function requires caller provide parameter.
- so parameter type is `xxx | undefined` and not `?: xxx`

Reply:
There are no required argument in this function?

Reply:
Updated, PTAL again

```ts
export const validateSignInMethods = (signInMethods: SignInMethods | undefined) => {
    …
}
```